cime_config/allactive/config_pesall.xml

```diff
@@ -2627,7 +2627,7 @@
     </mach>
   </grid>
   <grid name="a%ne30np4.pg2_l%.+_oi%EC30to60E2r2.+w%wQU225EC30to60E2r2">
-    <mach name="anvil">
+    <mach name="anvil|chrysalis">
```

Comment:
@erinethomas - I don't think we want this for both anvil and chrysalis, since they have different numbers of pes/node

Reply:
ah yes - I meant to remove all the PE configurations - I forgot to remove that option - thanks for noticing.

Reply:
should be corrected now.